opengrok-indexer/src/main/java/org/opengrok/indexer/configuration/Configuration.java

```diff
@@ -564,6 +566,7 @@ public Configuration() {
         setUserPageSuffix("");
         setWebappLAF("default");
         // webappCtags is default(boolean)
+        setAuthenticationTokens(new HashSet<>());
```

Comment:
// This list of calls is sorted alphabetically so please keep it.

Reply:
fixed